spec/controllers/resraurants_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, :type => :controller do
+  before(:all) do
+    @restaurants = FactoryBot.create_list(:restaurant, 3)
+  end
+
+  describe 'GET restaurants' do
```

Comment:
We usually call it `describe '#index'` (since it's obvious you're testing restaurant)